components/CloseButton/__tests__/CloseButton.test.js

```diff
@@ -10,9 +10,9 @@ describe('CloseButton', () => {
 
   it('should not be clickable when disabled', () => {
     const onClickMock = jest.fn();
-    const wrapper = mount(<CloseButton disabled onClick={onClickMock} />);
+    const wrapper = render(<CloseButton disabled onClick={onClickMock} />);
 
-    wrapper.find('button').simulate('click');
+    fireEvent.click(wrapper.container.querySelector('button'));
```

Comment:
Can you use `wrapper.queryByTestId`? I just added a `data-testid` to `Close Button`.